src/convertToObject.js

```diff
@@ -1,12 +1,21 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+
+  const arr = sourceString.trim().split(';');
+
+  for (let i = 0; i <= arr.length; i++) {
+    const elem = arr[i];
```

Comment:
I don't think that you need an index here, so you can use `for of` instead of common `for`